train.py

```diff
@@ -332,5 +350,12 @@ def get_lr(it):
     if iter_num > max_iters:
         break
 
+if init_from == "lora_finetune":
+    print('merging lora')
+    merge_lora(raw_model)
```

Comment:
Maybe save lora parameters in a standalone file?

Reply:
Agreed, the checkpoints actually already have the lora parameters in them (the parameterization is computed whenever the weights are referenced, including during exports). Saving them off to the side could enable hot swapping loras for different tasks at some point if folks are interested in that feature.